apps/wfo-ui/pages/tasks/new/index.tsx

```diff
@@ -1,28 +1,17 @@
-import React from 'react';
+import { useEffect } from 'react';
```

Comment:
The linter doesn't seem to pick it up because its in the package folder. It does give an unused error if I add it. I'm going to leave it for now because I don't wat to mess with the eslint config to much